src/ol/uri.js

```diff
@@ -0,0 +1,17 @@
+goog.provide('ol.uri');
+
+
+/**
+ * Appends query parameters to a URI.
+ *
+ * @param {string} uri The original URI, which may already have query data.
+ * @param {!Object} params An object where keys are URI-encoded parameter keys,
```

Comment:
Changed back into leaving keys untouched as this is the existing behaviour with `goog.uri.utils.appendParamsFromMap`.